integrations/src/test/resources/io/jenkins/plugins/casc/testJob2.groovy

```diff
@@ -1,12 +1,6 @@
 package io.jenkins.plugins.casc
 
 job('testJob2') {
-    scm {
-        git('git://github.com/quidryan/aws-sdk-test.git')
```

Comment:
JobDSL complains about git version number is too low.

Reply:
do we need to bump the git version instead then?

Reply:
it's inside the integrations module so 🤷‍♂️ We don't need to have more tests added when it is not needed 😄